afs-core/src/main/java/com/powsybl/afs/TaskMonitor.java

```diff
@@ -228,7 +226,7 @@ public boolean equals(Object obj) {
      * Return the complete state of tasks related to a project
      *
      * @param projectId related project
-     * @return
+     * @return description missing
```

Comment:
The `description missing` occurrences in this class should be replaced by real descriptions.

Reply:
done